tests/parsers/test.locale-messagesjson.js

```diff
@@ -92,22 +92,20 @@ describe('LocaleMessagesJSONParser', () => {
     expect(warnings[1].message).toEqual(messages.MISSING_PLACEHOLDER.message);
   });
 
-  it('should be invalid if bad message name', () => {
+  it('should not be an error if bad message name', () => {
```

Comment:
I'm a bit unsure about the "if bad messagename" from this test description, how about "should not be invalid on non alphanumeric message name"?